{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入正确账号", "status": "passed", "attachments": [{"name": "正确账号", "source": "3aa2ec38-168b-426f-966a-a92379d50a01-attachment.attach"}], "start": 1651654658578, "stop": 1651654658578}, {"name": "输入错误密码", "status": "passed", "attachments": [{"name": "错误密码", "source": "6ae63c09-09e3-4be8-b740-91fc3036504c-attachment.attach"}], "start": 1651654658614, "stop": 1651654658614}, {"name": "输入正确验证码", "status": "passed", "attachments": [{"name": "正确验证码", "source": "0cc48306-1a0f-4beb-a170-0fdb2a9fc7c9-attachment.attach"}], "start": 1651654658659, "stop": 1651654658659}], "attachments": [{"name": "截图", "source": "00258afe-0d36-4245-a879-f2e14c682d9c-attachment.png", "type": "image/png"}], "start": 1651654658578, "stop": 1651654661007, "uuid": "afa503f8-2928-42ef-848f-6b45a899564d", "historyId": "8ec91ebb07cb2b45a38478eca1be69f5", "testCaseId": "a86650c2206e92995fa9f8bad029494f", "fullName": "test_two.Test#test_06", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "xxx测试"}, {"name": "story", "value": "登录"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "6940-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}